packages/date-parser/src/result.js

```diff
@@ -0,0 +1,63 @@
+import { merge } from 'lodash';
+
+const convertToTargetTimezone = (parsedComponent) => {
+  if (!parsedComponent) return null;
+  const offset = parsedComponent.get('timezoneOffset');
+
+  return parsedComponent.dayjs().utcOffset(offset);
+};
+
+export default class PostResult {
```

Comment:
not sure why just not `Result`? Or `HResult` to indicate the final 'Holistics' result

Reply:
That's my first thought too : )) But still want to use PostResult to indicate this is the post result of the chrono parsing.

Reply:
why do we need to indicate that this is the result post-chrono-parsing?
* if it's for internal readability, then I believe it's enough to name the variable `post_chrono_result` during the processing execution.
* otherwise we should try to make the end result as friendly and intuitive to the call sites as possible.

(Also, just a soft reminder that this is an open-source package nha anh)

Reply:
Yeah, agree. I changed the variable name to `postChronoResult` and the class to `Result`